src/pages/en/meetings/2025-09-24-autogenerar-o-no-autogenerar.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: ../../../layouts/MeetingLayout.astro
+date: 2025-09-24 18:30
+title: "Building a Ruby Gem for OpenFGA: autogenerate or not?"
+location: "<a href='https://www.guruwalk.com/' target='_blank'>GuruWalk</a> Calle Guillem de Castro 9, 7th floor, Valencia 46007"
+location_map: "https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3080.0462490980685!2d-0.3831354867972076!3d39.46828387148948!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0xd604f4971596063%3A0xc7a87c4de358c8a7!2sC%2F%20Guillem%20de%20Castro%2C%209%2C%20Extramurs%2C%2046007%20Valencia!5e0!3m2!1ses!2ses!4v1749571813939!5m2!1ses!2ses"
+sponsor:
+  name: GuruWalk
+  url: https://www.guruwalk.com
+  logo: guruwalk.png
+  logo_width: "200px"
+---
+
+![Creating a Ruby gem for OpenFGA!](../../images/logos/openfga.png)
+
+We are back after summer with reacharged batteries! 🔋
+
+In this chance, [Carla Urrea Stabile](https://www.linkedin.com/in/carlastabile/) 
```

Comment:
I would use the word _"edition"_ instead of _chance_. Chance seems to be a direct translation from the spanish word.